__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const app = require("../app.js");
+const request = require("supertest")
+const db = require("../db/connection.js")
+const testData = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+
+afterAll(() => {
+    db.end();
+  });
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+describe("ALL *", () => {
```

Comment:
I like how you've split this up because it's an error not specific to the /api/topics endpoint